app/src/main/res/values-it/strings.xml

```diff
@@ -300,6 +300,7 @@
         <item quantity="one">Indirizzo e-mail bloccato</item>
         <item quantity="other">Indirizzi e-mail bloccati</item>
     </plurals>
+    <string name="manageSignatures">Gestire le mie firme</string>
```

Comment:
Aks @valentinperignon about the conjugation of that sentence

Reply:
@valentinperignon sent me the new string.